client/aws/errors.go

```diff
@@ -0,0 +1,5 @@
+package aws
+
+const (
+	KeyPairDuplicate = "InvalidKeyPair.Duplicate"
```

Comment:
IMO this is no error. This is some format/identifier which is used to detect an error.